docs/source/conf.py

```diff
@@ -173,12 +174,33 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 
+version_match = os.environ.get("READTHEDOCS_VERSION")
+
+# If READTHEDOCS_VERSION doesn't exist, we're not on RTD
+if not version_match:
+    version_match = "dev"
+
+# If it is an integer, we're in a PR build and the version isn't correct.
+elif version_match.isdigit():
+    version_match = "dev"
+
+# If it's "latest" → change to "dev" (that's what we want the switcher to call it)
+elif version_match == "latest":
+    version_match = "dev"
+
+else:
+    version_match = "stable"
+
+if version_match == "dev":
+    version = "dev"
+
 # documentation.
 html_theme_options = {
     "logo": {
         "image_light": "../../logo/logo.svg",
         "image_dark": "../../logo/logo_white.svg",
     },
+    "navbar_start": ["navbar-logo", "navbar-version"],
```

Comment:
is having the version on the left with the logo not redundant with having it displayed on the right in the switcher?